src/pages/login/login.css

```diff
@@ -0,0 +1,105 @@
+.login-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    background-color: var(--background-primary);
+    color: var(--font-primary);
+}
+
+.login-card {
+    display: flex;
+    background-color: var(--background-secondary);
+    border-radius: 7px;
+   
+}
+
+.login-hero-img {
+    width: 100%;
+    height: 25rem;
+}
+
+.login-main {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    padding: 2rem;
+    margin-left: 2rem;
+
+}
+
+.login-header {
+    margin-bottom: 1rem;
+    font-size: 2rem;
+    font-weight: 700;
+}
+
+.input-group {
+    display: flex;
+    margin: 1rem 0rem 0.5rem 0rem;
+    border-bottom: 2px solid var(--font-primary);
+    width: 15rem;
+
+}
+
+.input-group:focus-within {
+    border-bottom: 2px solid var(--primary);
+    transition: 0.4s;
+    transition-timing-function: linear;
+}
+
+.input-group:focus-within .input-svg {
+    fill: var(--primary);
+    transition: 0.5s;
+}
+
+.input-svg {
+    display: flex;
+    align-items: center;
+    fill: var(--font-primary);
+}
+
+.input-group input {
+    border: none;
+    padding: 0.3rem;
+    font-size: 0.8rem;
+    width: 100%;
+    color: var(--font-primary);
+    background-color: var(--background-secondary);
+}
+
+.input-group input:focus {
+    outline: none
+}
+
+.input-field {
+    margin: 0.5rem 0rem 0.9rem 0rem;
+}
+
+.input-check {
+    display: flex;
+    justify-content: space-between;
+}
+
+.input-check-left {
+    display: flex;
+    align-items: center;
+    gap: 2px;
+}
+
+label {
+    font-size: 0.7rem;
+}
+
```

Comment:
Don't target the elements directly, as this will effect globally.
Use a class instead